src/transport_picoquic.cpp

```diff
@@ -756,7 +771,7 @@ PicoQuicTransport::checkTxData()
 void
 PicoQuicTransport::sendTxData(StreamContext* stream_cnx, [[maybe_unused]] uint8_t* bytes_ctx, size_t max_len)
 {
-    if (bytes_ctx == NULL) {
+    if (bytes_ctx == nullptr) {
```

Comment:
```suggestion
    if (!bytes_ctx) {
```
Similar suggestion throughout, but this is a personal preference. Just feels weird to see NULL/nullptr everywhere.

Feel free to ignore this kind of comment :)